code/modules/mob/living/life.dm

```diff
@@ -2,12 +2,7 @@
 	set waitfor = FALSE
 	set invisibility = 0
 
-	if(HAS_TRAIT(src,TRAIT_DIGINVIS)) //AI unable to see mob
-		if(!digitaldisguise)
-			src.digitaldisguise = image(loc = src)
-		src.digitaldisguise.override = 1
-		for(var/mob/living/silicon/ai/AI in GLOB.ai_list)
-			AI.client?.images |= src.digitaldisguise
+	SEND_SIGNAL(src, COMSIG_LIVING_LIFE, seconds, times_fired)
```

Comment:
What is `COMSIG_LIVING_LIFE` for? It's added in this PR and not used anywhere as far as I can tell.

Reply:
It was added the same time these changes were made. It's currently unused in beecode, but probably nice to have.

Reply:
> It was added the same time these changes were made.

Do you mean the PRs you're porting? Because if yes, I can't find where the changes were made, could you maybe point it out exactly?

Reply:
No, just around the same time period on TG. I figured I might as well toss it in.